docs2/site/docs/migrations/migration4.md

```diff
@@ -11,7 +11,13 @@
 
 ## Breaking Changes
 
-* `NameConverter` and `SchemaFilter` have been removed from `ExecutionOptions` and are now properties on the `Schema`.
+* `NameConverter`, `SchemaFilter` and `FieldMiddleware` have been removed from `ExecutionOptions` and are now properties on the `Schema`.
+  These properties can be set in the constructor of the `Schema` instance, or within your DI registration delegate, or at any time before
```

Comment:
What is "DI registration delegate" ?

Reply:
```cs
Services.AddSingleton<ISchema>(
    //this delegate -- I don't know what to call it
    serviceProvider => {
        var schema = new Schema();
        schema.Query = serviceProvider.GetRequiredService<MyQuery>();
        schema.FieldMiddleware.Use(serviceProvider.GetRequiredService<MyMiddleware>());
    }
);
```
I'm open to any suggestions you have as to the documentation here.

Reply:
I used to call it _composition root_: https://blog.ploeh.dk/2011/07/28/CompositionRoot/

Reply:
Uhh...ok so this???
```suggestion
  These properties can be set in the constructor of the `Schema` instance, or within your DI composition root, or at any time before
```

Reply:
Yes.